First addition of visdag content.

otherorb opened the pull request:
Submitting the first pull request to origin.

rbeyer commented on .telemetry/id.yaml:
```diff
@@ -0,0 +1 @@
+instance_id: b4217c06-71cb-4406-8e5e-f9652868b855
```
This looks ephemeral, should it be in the repo?

otherorb replied:
Removed.

rbeyer commented on README.md:
```diff
@@ -1,2 +1,132 @@
-# visds
-VIPER VIS Data System
+<<<<<<< Updated upstream
```
There are conflict markers in this file, please clear.

otherorb replied:
Cleared.

rbeyer commented on setup.py:
```diff
@@ -0,0 +1,33 @@
+<<<<<<< Updated upstream
```
There are conflict markers in this file, please clear.

otherorb replied:
Cleared.

rbeyer commented on visdag/ops/__init__.py:
```diff
@@ -0,0 +1 @@
+
```
This file should exist and be empty or have contents, but a single blank line is weird.

otherorb replied:
Emptied file.

rbeyer commented on visdag/schedules/__init__.py:
```diff
@@ -0,0 +1 @@
+
```
blank line

otherorb replied:
Fixed.

rbeyer commented on visdag/visds_config.yaml:
```diff
@@ -0,0 +1,11 @@
+      db_config:
+        - DB_TYPE=postgresql
+        - POSTGRES_USER=viper
+        - POSTGRES_PASSWORD=viper
```
This file should not be in the repo.  You could include a visdag/visds_config.example.yml in the repo to show people what is expected, but the install instructions should instruct a user to create their own visdag/visds_config.yml from the example file.

otherorb replied:
Missed this file when changing to .env configuration. Removed file. 

A TODO is to update the install instructions. I want to get this PR done first, so will not add a new file yet.

rbeyer commented on visdag/watcher.json:
```diff
@@ -0,0 +1,262 @@
+[
```
Why is this file in the repo?

otherorb replied:
Junk file left over from a git add on the directory. Removed.

rbeyer commented on visdag_tests/__init__.py:
```diff
@@ -0,0 +1 @@
+
```
Blank line.

otherorb replied:
Okay, this is weird. I didn't touch any of the files in visdag_tests, so it looks like dagster is creating these __init__.py files with an empty line. Not sure why. Anyway, that's fixed.

rbeyer commented on visdag_tests/test_assets.py:
```diff
@@ -0,0 +1 @@
+
```
Blank line.

otherorb replied:
Fixed.

rbeyer reviewed (changes requested):
Also seems like a lot of redundant versions of watch_cam.

otherorb commented:
Note that I removed the watchcam files for now. The one useful version, rewritten for use as an asset_sensor and will be added back in with the new asset_sensor branch.
